hw/ip/otbn/dv/otbnsim/test/simple/add/add.s

```diff
@@ -0,0 +1,20 @@
+/* Copyright lowRISC contributors. */
+/* Licensed under the Apache License, Version 2.0, see LICENSE for details. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+    addi    x10, x0, 1
```

Comment:
Is there a reason why these snippets are indented?

Reply:
Emacs asm-mode's default style :-) But I accept this might not be a very good reason! I quickly looked at our [ASM style guide](https://docs.opentitan.org/doc/rm/asm_coding_style/#indentation) and it looks like maybe these should be 2-space indented. I'll change to that.

Reply:
I was more thinking about not indented at all?

Reply:
Well, I suppose we might have labels and jumps in some of the examples, so I guess it makes sense to keep the indentation for uniformity. Obviously, I don't have any strong feelings: would you prefer me to remove it?

Reply:
I'd personally remove the indentation if there's no label, but fine either way.

Reply:
Let's keep it like this for now (to avoid waiting for CI again!): very happy to change in future.